utils/connection.py

```
@@ -1,12 +1,18 @@
 import streamlit as st
-import os
+import os, datetime
+from dateutil.relativedelta import relativedelta
 from dotenv import load_dotenv
 
+def getCorrectTime():
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)